### PR TITLE
Página de criação de companies de teste

### PR DESCRIPTION
**Pull request by edumoreira1506**
## Contexto
Até então estávamos usando [essa página](https://dashboard.pagar.me/#/signup) para cadastrar companies de teste, essa página vai ser descontinuada e por isso estamos migrando esse formulário pra cá.

## Checklist
- [x] Consegue cadastrar uma company pelo formuláio
- [x] Depois de cadastrado e o e-mail confirmado, consegue fazer login
- [x] Quando a API retorna erros os erros são exibidos
- [x] Validações no front 

## Screenshots
image

### CODE REVIEW

File: packages/pilot/src/pages/Account/index.js [hunk 1]
```diff
@@ -40,6 +41,10 @@ const AccountArea = ({ t }) => (
             component={Login}
             path="/account/login"
           />
+          <Route
+            component={CompanySignUp}
+            path="/account/register"
```
Comment (zignd) [inline packages/pilot/src/pages/Account/index.js:46]: O que você acha de mudar essa rota para algo mais específico? Tipo `/test-company`. O mudar o nome do componente para refletir o nome da rota também.

File: packages/pilot/src/pages/Account/CompanySignUp/buildParamsErrors.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  }
+
+  return error.response.errors.map(item => ({
+    code: item.code,
+    message: item.message,
+    parameter_name: item.parameter_name,
+  }))
+}
+
+export default buildParamErrors
```
Comment (zignd) [inline packages/pilot/src/pages/Account/CompanySignUp/buildParamsErrors.js:13]: Será que vale a pena mover esse código para os formatters? Pelo visto tem outras ocorrências dele por aí também né?

File: packages/pilot/src/pages/Account/CompanySignUp/index.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        'X-Live': 0,
+      },
+    }
+
+    try {
+      const response = await pagarme
+        .client.company.create(sdxEnvironmentOptions, companyData)
+      const companyIsCreated = response?.object === 'company' ?? false
+
+      if (companyIsCreated) {
```
Comment (zignd) [inline packages/pilot/src/pages/Account/CompanySignUp/index.js:28]: :eyes:

File: packages/pilot/src/locales/pt/translations.json [hunk 4]
```diff
@@ -2078,5 +2082,8 @@
   "select_version_confirm_title": "Mudança de versionamento",
   "select_version_confirm_text": "Tem certeza de que deseja alterar a versão da API para a sua conta? Isso pode afetar a sua integração. Para saber mais sobre isso",
   "select_version_confirm_link": "clique aqui.",
-  "select_version_confirm_version": "Versão da API do modo {{env}}: {{version}}"
+  "select_version_confirm_version": "Versão da API do modo {{env}}: {{version}}",
+  "company_sign_up": "Cadastro Pagar.me",
+  "register": "Cadastrar",
+  "created_account": "Conta criada com sucesso!"
```
Comment (zignd) [inline packages/pilot/src/locales/pt/translations.json:2088]: Seria legal informar que um email de confirmação será enviado.

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 3a23dc65b8f4bc377370e0b7ad2fe12c83edab4b
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 154e98168e3ee958d6231ec9fd1ee8b2a6c20415
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 7d465cc918f2e4199615552e421c4cbc832d3f1f
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 3a23dc65b8f4bc377370e0b7ad2fe12c83edab4b
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 154e98168e3ee958d6231ec9fd1ee8b2a6c20415
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 7d465cc918f2e4199615552e421c4cbc832d3f1f
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 3a23dc65b8f4bc377370e0b7ad2fe12c83edab4b
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 154e98168e3ee958d6231ec9fd1ee8b2a6c20415
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 7d465cc918f2e4199615552e421c4cbc832d3f1f
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 3a23dc65b8f4bc377370e0b7ad2fe12c83edab4b
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 154e98168e3ee958d6231ec9fd1ee8b2a6c20415
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 7d465cc918f2e4199615552e421c4cbc832d3f1f
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: packages/pilot/src/pages/Account/index.js (hunk 1 above)
Comment (edumoreira1506) [reply packages/pilot/src/pages/Account/index.js]: Feito!

**Review by zignd — APPROVED** [review]

**Review by julianasobreira — APPROVED** [review]

**Pull request merged**